client/ionic/src/App.test.tsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
+import { IntlProvider } from 'react-intl';
+import { language, messages } from './i18n/i18n';
```

Comment:
Nit, should we call this locale instead of language? Since that's how it's referred to on line 9.
```suggestion
import { locale, messages } from './i18n/i18n';
```

Reply:
Fair point, I'll not block this though. Feel free to create a cleanup PR later .

Reply:
So the browser provides a composite field with language and optional region code suffixed.  Together I think these are considered the locale.  Since we are only chopping off the language prefix I called it language.  But I am happy to make the change if that clarifies.